src/workerd/io/worker.c++

```diff
@@ -1928,9 +1952,12 @@ void Worker::handleLog(jsg::Lock& js,
     KJ_ASSERT(formatLogVal->IsFunction());
     auto formatLog = formatLogVal.As<v8::Function>();
 
+    const char* levelStr = logLevelToString(level);
     args[length] = v8::Boolean::New(js.v8Isolate, colors);
+    args[length + 1] = v8::Boolean::New(js.v8Isolate, structuredLogging);
+    args[length + 2] = jsg::v8StrIntern(js.v8Isolate, levelStr);
```

Comment:
You could likely get away with just passing `levelStr` conditionally. If the value is passed as `undefined`, then assume `structuredLogging` is false, otherwise we assume `structuredLogging` is true.

Reply:
Yeah I could but that sounds ugly and odd doesn't it? why not convey intent directly?

Reply:
Why odd? It's fairly idiomatic in JavaScript.

```c++
args[length + 1] = js.undefined();
if (structuredLogging) {
  args[length + 1] = js.strIntern(levelStr);
}
```

Then in JS

```js
if (level !== undefined) {
  // structured logging enabled
} else {
  // structured logging disabled
}
```